### PR TITLE
ATsurkanu branch

### PR DESCRIPTION
**Pull request by ATsurkanu**
Created base entites, but they shoud be reviewed later with all team.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,6 +18,28 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+
+        <dependency>
```
Comment (chapai) [inline pom.xml]: Looks like new dependencies are unused in this PR.

File: src/main/java/com/AristoPets/Hello/entity/Advertisement.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    private String club;
```
Comment (chapai) [inline src/main/java/com/AristoPets/Hello/entity/Advertisement.java]: why not keep all properties together?

File: src/main/java/com/AristoPets/Hello/entity/Animal.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    public String getAdditionalInformation() {
+        return additionalInformation;
+    }
+
+    public void setAdditionalInformation(String additionalInformation) {
+        this.additionalInformation = additionalInformation;
+    }
+
+    private String additionalInformation;
```
Comment (chapai) [inline src/main/java/com/AristoPets/Hello/entity/Animal.java]: same

**Comment by chapai** [comment]
@AristoPets/backend PTAL

**Comment by chapai** [comment]
Some extra new lines before class end at` Advertisment.java` and `Animal.java`.
Overall, LGTM! Thanks!

**Comment by ATsurkanu** [comment]
TY. So, can we merge?

**Pull request merged**